src/js/popup.js

```diff
@@ -102,7 +103,7 @@ class UI {
     if (this.isOpenReviewUrl(url)) return this.getOpenReviewInfo(url);
   }
   parseArXivId(str) {
-    const paperId = str.match(/\d+.\d+/);
+    const paperId = str.match(ARXIV_ID_REGEX)?.[0] || str.match(ARXIV_OLD_ID_REGEX)?.[0];
     return paperId;
   }
```

Comment:
```suggestion
  // ref: https://info.arxiv.org/help/arxiv_identifier.html
  // e.g. (new id format: 2404.16782) | (old id format: hep-th/0702063)
  parseArXivId = (str) => str.match(/(\d+\.\d+$)|((\w|-)+\/\d+$)/)?.[0];
```

Reply:
I have simplified the code and included a reference to the official arXiv document. Additionally, I added "$" to prevent matching (abs|pdf)/\d+.